<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1451</width>
    <height>828</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>93</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Heap_sort">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>Heapsort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Merge_sort">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>450</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>Mergesort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Quick_sort">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>Quick sort</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>1401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>471</width>
      <height>581</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>70</y>
      <width>821</width>
      <height>641</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>223</x>
      <y>150</y>
      <width>251</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>223</x>
      <y>450</y>
      <width>251</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Size array</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>740</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Time Sort</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_3">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>730</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>680</y>
      <width>471</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Input">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>700</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>binarySearch</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>223</x>
      <y>700</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <zorder>listWidget_3</zorder>
   <zorder>listWidget_2</zorder>
   <zorder>listWidget</zorder>
   <zorder>lineEdit</zorder>
   <zorder>Heap_sort</zorder>
   <zorder>Merge_sort</zorder>
   <zorder>Quick_sort</zorder>
   <zorder>line</zorder>
   <zorder>graphicsView</zorder>
   <zorder>textEdit</zorder>
   <zorder>textEdit_2</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>listWidget_4</zorder>
   <zorder>Input</zorder>
   <zorder>lineEdit_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1451</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
